### ISSUE TITLE
JVMTest

### ISSUE THREAD

**Issue by neo-expert** [guxingke/mini-jvm#20]
Maybe you can use my JVMTest.java Application which tests every Bytecode-Instruction (except INVOKEDYNAMIC):
https://gitlab.com/neoexpert/jvm/-/blob/184d80e18d6cfb130146604ea3f02f64f2843654/JVMTest.java

When your JVM can run this, then it is almost complete.

**Comment by guxingke** [comment]
thanks ,i found it works well except native print method.

**Comment by neo-expert** [comment]
Yeah this method will only be invoked, when JVM is not correctly implemented. You can use it to deliver error codes.

**Comment by guxingke** [comment]
wow , I want to check again. 😂
